src/megaapi_impl.cpp

```diff
@@ -10197,6 +10206,16 @@ void MegaApiImpl::account_details(AccountDetails*, error e)
     fireOnRequestFinish(request, megaError);
 }
 
+void MegaApiImpl::querybandwidthquota_result(int code)
+{
+    if(requestMap.find(client->restag) == requestMap.end()) return;
+    MegaRequestPrivate* request = requestMap.at(client->restag);
+    if(!request || (request->getType() != MegaRequest::TYPE_QUERY_BANDWIDTH_QUOTA)) return;
+
```

Comment:
If we return in the `Command::procresult()` an `API_EINTERNAL` (or any other error code), I think we should not set the flag in the request unless the code is `API_OK`